posixlib/src/main/resources/scala-native/time.c

```diff
@@ -93,6 +93,18 @@ size_t scalanative_strftime(char *buf, size_t maxsize, const char *format,
 
 char **scalanative_tzname() { return tzname; }
 
-long scalanative_timezone() { return timezone; }
+long scalanative_timezone() {
+#if !defined(__FreeBSD__)
```

Comment:
I am probably in the minority and would have to check the linux
practice, but I have found it easier to follow (fewer gymnastics) 
the positive `defined` rather than the negative `!defined`.  That is
```
#if defined(__FreeBSD__)
           return 0;
#else 
            return timezone
#endif
```
Not only is it easier at present to see what is happening in the __FreeBSD__
case, it is also easier to know where the code goes when someone
down the line wants to add __SomeWierdBehavior__.

I think there are at least two cases of this idiom here.

You might want to see what someone from Scala Center  so that
my possibly minority position does not cause you to waste effort.

I mention it here for consideration for other reviewers. We all are going
to have to live with and maintain this code for a long time. 

I have not checked with what prior art in that file, if any, does.

Reply:
If it were up to me I would have written it in the positive form as you suggested, but then I thought that perhaps the project prefers to have the "normal" code first and the "exception" later.
This comments applies to all other points where you found a negative form as well.

Reply:
Let's change it to the positive condition as LeeTibbert suggested.